GestureHandler.js

```diff
@@ -383,7 +407,7 @@ function createNativeWrapper(Component, config = {}) {
       // filter out props that should be passed to gesture handler wrapper
       const gestureHandlerProps = Object.keys(this.props).reduce(
         (props, key) => {
-          if (key in NATIVE_WRAPPER_PROPS_FILTER) {
+          if (~NATIVE_WRAPPER_PROPS_FILTER.indexOf(key)) {
```

Comment:
```suggestion
          if (NATIVE_WRAPPER_PROPS_FILTER.indexOf(key) !== -1) {
```

~ is bitwise not, right? It works but it's very unclear imho.